x-pack/plugins/enterprise_search/public/applications/elasticsearch/components/layout/page_template.tsx

```diff
@@ -14,6 +14,7 @@ import { useEnterpriseSearchNav } from '../../../shared/layout';
 import { SendEnterpriseSearchTelemetry } from '../../../shared/telemetry';
 
 export const EnterpriseSearchElasticsearchPageTemplate: React.FC<PageTemplateProps> = ({
+  access,
```

Comment:
question: Have you thought about pulling the access value from a central spot like a singleton or maybe Kea, instead of passing it down as props?

Reply:
I knew we'd get a better suggestion in review! Thanks @sphilipse ❤️. Pulling this from something like `AppLogic` would probably make more sense.